pymatgen/io/lobster/inputs.py

```diff
@@ -688,13 +691,13 @@ def standard_calculations_from_vasp_files(
             "onlycohpcoop",
             "onlycohpcoopcobi",
             "onlymadelung",
-        ]:
+        }:
             raise ValueError("The option is not valid!")
 
         lobsterin_dict: dict[str, Any] = {
-            # this basis set covers most elements
+            # This basis set covers most elements
```

Comment:
```suggestion
            # This basis set covers elements up to Lr, Z = 103
```

Reply:
I should trust you as a LOBSTER developer 😄 , but I still checked the manual and confirmed this: 
> If you wish to override the basis set selected by default, you
may do so with this keyword. It currently supports bunge
(for elements up to Xe), koga and pbeVaspFit2015 (up to
Lr; yes, that is Z = 103).

Reply:
I copied it from the manual haha